### PR TITLE
tests: Make build fail if Zotonic fails to start

### LINKED ISSUE #1302: Zotonic runtests returns 0 in case of failure
We currently have (on 0.17.0):

```bash
bin/zotonic runtests

# …

3:55:59.757 [info] Starting reporters with []
13:55:59.758 [info] Application exometer_core started on node zotonic001_testsandbox@ginger
13:55:59.761 [info] Application exometer started on node zotonic001_testsandbox@ginger
13:55:59.763 [info] Application epgsql started on node zotonic001_testsandbox@ginger
** Found 0 name clashes in code paths
13:55:59.894 [error] Zotonic start error: no such file or directory: syslog.app
```

This still returns `0` (success) on Travis, probably due to https://github.com/zotonic/zotonic/blob/master/src/scripts/zotonic-runtests#L59.

### PR DESCRIPTION
**Pull request by ddeboer**
Fix #1302

### CODE REVIEW

File: src/zotonic.app.src [hunk 1]
```diff
@@ -18,6 +18,6 @@
                   depcache, mimetypes, webzmachine, z_stdlib,
                   poolboy, sendfile, public_key, ssl,
                   jobs, gproc, filezcache, s3filez,
-                  qdate
+                  qdate, bla
```
Comment (ddeboer) [inline src/zotonic.app.src]: This should cause a red build now! Let’s find out... :smirk:

**Comment by ddeboer** [comment]
Okay, this now works properly: 

screen shot 2016-06-09 at 09 04 01

**Comment by mworrell** [comment]
👍

**Comment by mworrell** [comment]
Thanks!

Also merged 99c2702e5ff8b0f3a659c24660a8134544905a60 into master

**Pull request merged**